pkg/kubelet/util/util_windows.go

```diff
@@ -123,3 +131,37 @@ func GetBootTime() (time.Time, error) {
 	}
 	return currentTime.Add(-time.Duration(output) * time.Millisecond).Truncate(time.Second), nil
 }
+
+type ReparseDataBufferHeader struct {
+	ReparseTag        uint32
+	ReparseDataLength uint16
+	Reserved          uint16
+}
+
+type ReparseDataBuffer struct {
+	Header ReparseDataBufferHeader
+	Detail [syscall.MAXIMUM_REPARSE_DATA_BUFFER_SIZE]byte
+}
+
+// IsUnixDomainSocket returns whether a given file is a AF_UNIX socket file
+func IsUnixDomainSocket(fileInfo os.FileInfo, filePath string) (bool, error) {
```

Comment:
fileInfo is not used in the function?

Reply:
In the Windows version of `IsUnixDomainSocket`, fileInfo parameter is not used. Only the filePath is used.

Reply:
for linux case, can get fileInfo inside this function instead of passing it as a parameter?

Reply:
Good point - I will add that.

Reply:
Addressed